.github/workflows/go.yml

```diff
@@ -0,0 +1,33 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
```

Comment:
Is it ok removing this blank line to follow the same pattern of `on: push: `?

Reply:
I think it's ok. I gonna test it

Reply:
Done